modules/nf-core/custom/filterdifferentialtable/main.nf

```diff
@@ -3,9 +3,7 @@ process CUSTOM_FILTERDIFFERENTIALTABLE {
     label 'process_single'
 
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pandas:1.5.2' :
-        'biocontainers/pandas:1.5.2' }"
+    container "community.wave.seqera.io/library/pandas_python:24935c20d1a97271"
```

Comment:
Please add the singularity container too (there is a specific format that is available somewhere but I can't find immediately, remind me on Monday).

Reply:
Hi @SPPearce, please point to me towards the format that you are expecting here. Having the container with this format works for singularity too.